code/core/src/core-server/dev-server.ts

```diff
@@ -165,6 +166,34 @@ export async function storybookDevServer(options: Options) {
         return;
       }
     });
+
+    app.get('/manifests/components.html', async (req, res) => {
```

Comment:
I think it would be nice to also support `'/manifests/components'`, without the html extension. just call `get()` two times with the same handler.